dynawo/sources/Modeler/DataInterface/DYNDataInterface.h

```diff
@@ -38,6 +38,18 @@ class DataInterface {
    */
   virtual ~DataInterface() { }
 
+  /**
+   * @brief Determines if variant managing is supported
+   * @returns @b true if network variant is supported, @b false if not
+   */
+  virtual bool canUseVariant() const = 0;
+
+  /**
+   * @brief Choose which network variant to use
+   * @param variantName the name of the variant to use
+   */
+  virtual void useVariant(const std::string& variantName) = 0;
```

Comment:
I would propose setVariant or selectVariant to be clear on the fact that this is a setter

Reply:
OK